src/main/groovy/org/pih/warehouse/importer/ProductAssociationExcelImporter.groovy

```diff
@@ -12,10 +12,18 @@ package org.pih.warehouse.importer
 import grails.gorm.transactions.Transactional
 import grails.util.Holders
 import org.grails.plugins.excelimport.AbstractExcelImporter
+import org.grails.plugins.excelimport.DefaultImportCellCollector
+import org.grails.plugins.excelimport.ExcelImportService
 import org.grails.plugins.excelimport.ExpectedPropertyType
 
 @Transactional
-class ProductAssociationExcelImporter extends AbstractExcelImporter {
+class ProductAssociationExcelImporter extends AbstractExcelImporter implements DataImporter {
+
+    static cellReporter = new DefaultImportCellCollector()
+
+    ExcelImportService excelImportService
+    @Delegate
+    ProductAssociationImportDataService productAssociationImportDataService
```

Comment:
I believe there is an indent missing between `ExcelImportService` and the `@Delegate` annotation.
Would be good to fix at least that.

Reply:
At first I did not understand what do you mean by indent and thought that maybe github has formatted the code for you in some weird way because I don't see any wrong indentation (tabs from the left).
But now I understand that you meant spcaing between service bean declaration.

Sure I can fix that :D